sitegen.sln

```diff
@@ -2,7 +2,7 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sitegen", "src\Sitegen\Sitegen.csproj", "{A407BBF4-56E2-4720-80B0-D7B4AFD51F35}"
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Sitegen.Tests", "src\Sitegen.Tests\Sitegen.Tests.fsproj", "{B260256E-34CF-4ABA-8E55-A1C5FEA9E940}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Sitegen.Tests", "src\Sitegen.Tests\Sitegen.Tests.csproj", "{B260256E-34CF-4ABA-8E55-A1C5FEA9E940}"
```

Comment:
I ended up converting the test project introduced in #53 into plain C# after all. It was a specific feature in C# that was lacking in F# that I felt was so important that I didn't want to be without it - _inner classes_.

When writing unit tests, I like to be able to group related test methods into an inner class, for testing a particular semantic detail. Not being able to do this turned out to be a showstopper to me. While I _did_ like the F# experiment, I simply didn't want to be without this feature available in C#.

This does seem to be on the "approved backlog" for F# though, so who knows, maybe I'll end up converting the tests back to F# at some point. :grin: https://github.com/fsharp/fslang-suggestions/issues/277

Reply:
Oops, I forgot one change at this point: https://github.com/perlun/sitegen/pull/56/files#r676808497